srcpkgs/xfce4-weather-plugin/template

```diff
@@ -1,14 +1,14 @@
 # Template file for 'xfce4-weather-plugin'
 pkgname=xfce4-weather-plugin
 version=0.8.10
-revision=1
+revision=2
 lib32disabled=yes
 build_style=gnu-configure
 configure_args="--disable-upower"
 hostmakedepends="pkg-config intltool"
 makedepends="libxfce4ui-devel xfce4-panel-devel libxml2-devel libsoup-devel"
 depends="hicolor-icon-theme"
-short_desc="XFCE panel plagin to show temperature and weather"
+short_desc="XFCE panel plugin to show temperature and weather"
 maintainer="Juan RP <xtraeme@voidlinux.eu>"
 homepage="https://goodies.xfce.org/projects/panel-plugins/xfce4-weather-plugin"
 license="GPL-2"
```

Comment:
While we are at the fixing business please set this to the SPDX short identifier.

in this case it is `GPL-2.0-or-later`

Reply:
done, if it's okay to have them in the same commit